ndx-microscopy/ndx-meta.yaml

```
@@ -0,0 +1,8 @@
+name: ndx-microscopy
+version: 0.2.0
+src: https://github.com/catalystneuro/ndx-microscopy
+pip: https://pypi.org/project/ndx-microscopy/
+license: BSD-3
```

```suggestion
license: MIT
```

https://github.com/catalystneuro/ndx-microscopy/blob/main/LICENSE.txt